specs/withdrawals.md

```diff
@@ -237,3 +216,6 @@ contracts if desired.
 | `FINALIZATION_PERIOD` | `604_800` | seconds |
 
 This `FINALIZATION_PERIOD` value is equivalent to 7 days.
+
+[`WithdrawalTransaction` type]: https://github.com/kroma-network/kroma/blob/dev/packages/contracts/contracts/libraries/Types.sol#L131-L138
+[`OutputRootProof` type]: https://github.com/kroma-network/kroma/blob/dev/packages/contracts/contracts/libraries/Types.sol#L34-L40
```

Comment:
Would it be better to maintain the link with commit sha instead of the branch name, since branches can easily change?